README.md

```diff
@@ -1,49 +1,180 @@
+- Rebase the dependent PR when the base PR is updated.
+- Remove the merged PRs from the stack.
+- Split a PR / commit into multiple PRs / commits.
+- Reorder the PRs and commits in the stack.
 
-### For other platforms
-Download the latest av executable from the GitHub releases page on the av repository. Extract the archive and add the executable to your PATH.
+# Demo
 
+![Demo](resources/demo.gif)
 
-See rest of the instructions on
-[Aviator Stacked PRs quickstart](https://docs.aviator.co/aviator-cli).
+# Usage
 
-# Development setup
+> [!TIP]
+> Complete documentation is available at [docs.aviator.co](https://docs.aviator.co/aviator-cli).
 
-Install the latest version of Go from https://go.dev/doc/install.
+Create a new branch and make some changes:
 
-To run the command line:
+```sh
+$ av stack branch feature-1
+$ echo "Hello, world!" > hello.txt
+$ git add hello.txt
+$ git commit -m "Add hello.txt"
+```
 
+Create a PR:
+
+```sh
+$ av pr create
 ```
-go run ./cmd/av [subcommand/flags...]
+
+Create a new branch and make some changes. Create another PR that depends on the
+previous PR:
+
+```sh
+$ av stack branch feature-2
+$ echo "Another feature" >> hello.txt
+$ git add hello.txt
+$ git commit -m "Update hello.txt"
+$ av pr create
 ```
 
-# Release
+Visualize the PR stack:
+
+```sh
+$ av stack tree
+  * feature-2 (HEAD)
+  │ https://github.com/octocat/Hello-World/pull/2
+  │
+  * feature-1
+  │ https://github.com/octocat/Hello-World/pull/1
+  │
+  * master
+```
 
-To create a release, create a tag with the desired version and push to GitHub.
+Merge the first PR:
 
+```sh
+$ gh pr merge feature-1
 ```
-# Change the version as appropriate
-TAG="v0.0.0"
 
-git tag "$TAG"
-git push origin tags/"$TAG"
+Sync the stack:
+
+```sh
+$ av stack sync
+
+  ✓ GitHub fetch is done
+  ✓ Restack is done
+
+    * ✓ feature-2 f9d85fe
+    │
+    * master 7fd1a60
+
+  ✓ Pushed to GitHub
+
+    Following branches do not need a push.
+
+      feature-1: PR is already merged.
+
+    Following branches are pushed.
+
+      feature-2
+        Remote: dbae4bd Update hello.txt 2024-06-11 16:41:18 -0700 -0700 (2 minutes ago)
+        Local:  f9d85fe Update hello.txt 2024-06-11 16:43:41 -0700 -0700 (7 seconds ago)
+        PR:     https://github.com/octocat/Hello-World/pull/2
+
+  ✓ Deleted the merged branches
+
+    Following merged branches are deleted.
+
+      feature-1: f2335eec783b54226a7ab90f4af1c9b8309f8b61
+
 ```
 
-This will automatically trigger [Goreleaser](https://goreleaser.com/) (as part
-of the
-[`release.yml` workflow](https://github.com/aviator-co/av/blob/master/.github/workflows/release.yml))
-which will create a GitHub release and build and publish binaries to Homebrew
-and Scoop.
+# Installation
+
+`av` is available for macOS and Linux. In order to interact with GitHub, `av`
+requires an API token. If you have [GitHub CLI](https://cli.github.com/)
+installed, `av` will use the token from the GitHub CLI. Recommended to install
+both.
```

Comment:
```suggestion
```suggestion
installed, `av` will use the token automatically from the GitHub CLI. It is
recommended to install both.
```